invoice_comment_template/views/account_invoice_view.xml

```diff
@@ -1,12 +1,12 @@
-<openerp>
+<odoo>
   <data>
     <record model="ir.ui.view" id="invoice_form_add_comment">
       <field name="name">account.invoice.form.comment</field>
       <field name="model">account.invoice</field>
       <field name="inherit_id" ref="account.invoice_form"/>
       <field name="arch" type="xml">
 
-        <xpath expr="/form/sheet/notebook/page[3]" position="after">
+        <xpath expr="/form/sheet/notebook" position="inside">
           <page string="Comments" name="comments">
             <label string="The comments will be displayed on the printed document. You can load a predefined template, write your own text or load a template and then modify it only for this document." colspan="2"/>
```

Comment:
Labels are different in 10.0 from how they were in 8.0 (now they are bold); consider to change it to a <p> and maybe to wrap it in a group for appropriate padding from notebook tabs.